smurfs/src/components/App.css

```diff
@@ -1,28 +1,320 @@
 .App {
+  width: 100%;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
   text-align: center;
 }
 
-.App-logo {
-  animation: App-logo-spin infinite 20s linear;
-  height: 80px;
+.smurf-family {
+  position: absolute;
+  z-index: 400;
+  width: 800px;
+  margin: 0 auto;
+  left: 0;
+  right: 0;
+  top: 76%;
+  position: fixed;
+  user-select: none;
+  pointer-events: none;
+}
+
+.background2 {
+  width: 100%;
+  height: 50%;
+  background: rgb(223, 191, 52);
+  position: absolute;
+  left: 0;
+  top: 49%;
+  z-index: 200;
+  position: fixed;
+}
+
+.background {
+  width: 100%;
+  height: 50%;
+  background: rgb(233, 222, 67);
+  background: -moz-radial-gradient(
+    circle,
+    rgba(233, 222, 67, 1) 45%,
+    rgba(255, 209, 0, 1) 100%
+  );
+  background: -webkit-radial-gradient(
+    circle,
+    rgba(233, 222, 67, 1) 45%,
+    rgba(255, 209, 0, 1) 100%
+  );
+  background: radial-gradient(
+    circle,
+    rgba(233, 222, 67, 1) 45%,
+    rgba(255, 209, 0, 1) 100%
+  );
+  filter: progid:DXImageTransform.Microsoft.gradient(startColorstr="#e9de43",endColorstr="#ffd100",GradientType=1);
+  position: absolute;
+  left: 0;
+  top: 50%;
+  z-index: 200;
+  position: fixed;
 }
 
 .App-header {
-  background-color: #222;
-  height: 150px;
+  text-align: center;
+  width: 99%;
+  margin-top: 50px;
+  position: absolute;
+  z-index: 400;
+  user-select: none;
+  pointer-events: none;
+}
+
+.smurfs-logo {
+  width: 350px;
+}
+
+.title {
+  color: white;
+  font-family: 'Luckiest Guy', cursive;
```

Comment:
I need to steal this font - I remember it from the last time and you know its just as good and honestly I didn't even notice until I saw the name so that goes to show you how influential the context and environment that the typography is with is so important... love it